fcrepo-audit-triplestore/src/test/java/org/fcrepo/camel/audit/triplestore/integration/AuditSparqlIT.java

```diff
@@ -80,26 +94,50 @@
 
     private static final String AS_NS = "https://www.w3.org/ns/activitystreams#";
 
-    @EndpointInject(uri = "mock:sparql.update")
+    @EndpointInject("mock:sparql.update")
     protected MockEndpoint sparqlUpdateEndpoint;
 
-    @EndpointInject(uri = "mock:sparql.query")
+    @EndpointInject("mock:sparql.query")
     protected MockEndpoint sparqlQueryEndpoint;
 
-    @Produce(uri = "direct:start")
+    @Produce("direct:start")
     protected ProducerTemplate template;
 
+    @Autowired
+    private CamelContext camelContext;
+
+    @BeforeClass
+    public static void beforeClass() {
+        final String jmsPort = System.getProperty("fcrepo.dynamic.jms.port", "61616");
+        final Properties props = new Properties();
+        System.setProperty("triplestore.indexer.enabled", "true");
+        System.setProperty("indexing.predicate", "true");
```

Comment:
Just wondering if this property is needed here? It seems like it is used for solr indexing. `triplestore.indexing.predicate` is used for triple store indexing, but given that its not being set here and the test is passing it seems like we may not need that either